src/component/Photo/Photo.js

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import "./Photo.css";
+const Photo = () => {
+  return (
+    <div className="imgs">
+      <img src="img/circle.png" alt="" className="circular" />
+      <img src="img/person.png" alt="" className="person" />
+      <img src="img/react.png" alt="" className="react" />
+      <img src="img/star.png" alt="" className="star" />
+      <img src="img/Figma.png " alt="" className="figma" />
+      <img src="img/sttar.png " alt="" className="sttar" />

```

Comment:
![image](https://github.com/Leen-odeh12/Eduport_LMS/assets/81507185/7ea16451-adf8-43ff-b911-d1d50235a86e)

Reply:
done in next branch :)

Reply:
It's updated in the next branch